uber/templates/preregistration/attendee_donation_form.html

```diff
@@ -5,7 +5,7 @@
 <div class="masthead">
 </div>
 <div class="panel panel-default">
-{% if attendee.paid == c.NOT_PAID or attendee.overridden_price %}
```

Comment:
There's no clear reason to display the altered message telling them their registration price unless they haven't paid, though if someone disagrees please let me know.